backend/Dockerfile

```diff
@@ -10,8 +10,10 @@ COPY api/*.csproj api/
 COPY entities/*.csproj entities/
```

Comment:
For some reason now when I build the docker container it fails unless I first delete all the `bin` and `obj` folders of all of the projects...

Reply:
probably need to add those binaries to the dockerignore.

**/bin
**/obj

Reply:
What Devin said

Reply:
We already have the following on our dockerignore but that only matches a /bin folder under backend. 

```
[Bb]in/
[Oo]bj/
[Oo]ut/
```

We need

```
**/[Bb]in/
**/[Oo]bj/
**/[Oo]ut/
```